test/gtest/ucp/test_ucp_peer_failure.cc

```diff
@@ -147,10 +149,9 @@ void test_ucp_peer_failure::init() {
     wrap_errors();
 
     ucp_ep_params_t ep_params_mod = {0};
-    ep_params_mod.field_mask = UCP_EP_PARAM_FIELD_ERR_HANDLER_CB |
-                               UCP_EP_PARAM_FIELD_USER_DATA;
-    ep_params_mod.err_handler_cb = err_cb_mod;
-    ep_params_mod.user_data      = reinterpret_cast<void *>(uintptr_t(MAGIC));
+    ep_params_mod.field_mask = UCP_EP_PARAM_FIELD_ERR_HANDLER;
+    ep_params_mod.err_handler.cb = err_cb_mod;
+    ep_params_mod.err_handler.arg = reinterpret_cast<void *>(uintptr_t(MAGIC));
```

Comment:
maybe should use USER_DATA here as well, as in other tests?

Reply:
This is just for testing of `err_handler.arg` as well

Reply:
lets add comment for this?

Reply:
Ok